### PR TITLE
Refactor spotify model

### PR DESCRIPTION
**Pull request by robota-x**

### CODE REVIEW

File: server/models/spotifyPlaylist.js [hunk 1]
```diff
@@ -13,62 +13,59 @@ var spotifyApi = new SpotifyWebApi({
...
+
+parseSongsToUri = function(songList){
+  return songList.tracks.items.map(function(song) {
+    return song.uri;
+  });
+};
 
-searchForTracks = function (artistName){
-  return spotifyApi.searchTracks('artist:'+artistName)
+getTopArtistTrack = function (artistName){
```
Comment (JojSh) [inline server/models/spotifyPlaylist.js]: Should be plural (getTopArtistTracks)

File: server/routes/spotifyPlaylist.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 var router = express.Router();
 
-var getTracksAndCreatePlaylist = require('../modules/spotifyPlaylist.js').getTracksAndCreatePlaylist;
+var getTracksAndCreatePlaylist = require('../models/spotifyPlaylist.js').getTracksAndCreatePlaylist;
 
 router.get('/new', function(req, res){
   // songkick API uses res.body.festivalName to get arrayOfArtists
   //res.body.festivalName
-  getTracksAndCreatePlaylist(["Beyonce", "Coldplay"], req.user.userId, req.user.accessToken, "Glastonbury");
+  getTracksAndCreatePlaylist(["Arch Enemy", "In Flames", "Insomnium", "Eskimo Callboy", "The Prodigy", "NOFX", "Slipknot", "Foo Fighters", "Rise Against"], req.user.userId, req.user.accessToken, "Superhot Festival 4");
```
Comment (eripheebs) [inline server/routes/spotifyPlaylist.js:9]: lolols  @ your playlists:"P

**Comment by eripheebs** [comment]
:shipit:

**Comment by JojSh** [comment]
:shipit:

**Pull request merged**